src/handlers/inbound/consentRequests/{ID}.ts

```diff
@@ -66,7 +68,18 @@ async function patch (_context: unknown, request: Request, h: StateResponseToolk
 
   // don't await on promise to be resolved
   setImmediate(async () => {
-    await model.run()
+    try {
+      await model.run()
+    } catch (error) {
```

Comment:
where are we inspecting or passing the `error` to the client?
-> maybe w should also dump it to the log

Reply:
What kind of error are you thinking? I was assuming if the state machine didn't run it would just be a server error. The other cases of errors are handled in each individual transistion.

Reply:
in run method, in its global catch, we transit fsm to 'errored' state and then rethrow received exception so we got it here as an `error` parameter of catch. We should at least dump this error to log.

Reply:
ok. I forgot that I'm rethrowing the errors in trasistions. So I'll just log the error here and count on the transistions to send back PUT consentRequests/{ID}/error request.